cmd/lotus-pcr/main.go

```diff
@@ -467,24 +467,24 @@ func (r *refunder) Refund(ctx context.Context, tipset *types.TipSet, refunds *Mi
 	return nil
 }
 
-type repo struct {
+type Repo struct {
```

Comment:
(Private types returned by public functions. Not a real issue, but an easy fix to make the linter happy)